apps/et/et-cron-wa-task-for-expired-bfactions/et-cron-wa-task-for-expired-bfactions.yaml

```
@@ -6,7 +6,7 @@ spec:
   releaseName: et-cron-wa-task-for-expired-bfactions
   values:
     job:
-      image: hmctspublic.azurecr.io/et/cos:prod-ef1ba74-20250806110815 #{"$imagepolicy": "flux-system:et-cos"}
+      image: hmctspublic.azurecr.io/et/cos:pr-2706-7e3a09d-20250806084001 #{"$imagepolicy": "flux-system:demo-et-cron-wa-task-for-expired-bfactions"}
```

Just remembered, this shouldn't be changed as it's the base file

```suggestion
      image: hmctspublic.azurecr.io/et/cos:prod-ef1ba74-20250806110815 #{"$imagepolicy": "flux-system:et-cos"}
```